src/layouts/PageLayoutOrlando.astro

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
added page layout per city.  This enables us to keep a bunch of stuff common but create differences per city.